public/locales/bg/translation.json

```diff
@@ -105,20 +105,17 @@
         "syncing": "Синхронизиране",
         "unsync": "Десинхронизиране"
     },
-    "Downloading": "Сваляне",
```

Comment:
There are a lot of things being removed here by the `yarn i18next` command

Reply:
Oh I think it's because I removed that filter on a previous PR and didn't know I had to run `yarn i18next` again, those keys are not used anymore

Reply:
Yes